spec/facets/ProcessingEngineVersionRunFacet.json

```diff
@@ -0,0 +1,41 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://openlineage.io/spec/facets/1-0-0/ProcessingEngineVersionRunFacet.json",
+  "$defs": {
+    "ProcessingEngineVersionRunFacet": {
+      "allOf": [
+        {
+          "$ref": "https://openlineage.io/spec/1-0-3/OpenLineage.json#/$defs/RunFacet"
+        },
+        {
+          "type": "object",
+          "properties": {
+            "engine_version": {
+              "description": "Processing engine version. Might be Airflow or Spark version.",
+              "type": "string",
+              "example": "2.5.0"
+            },
+            "engine_name": {
```

Comment:
I think it's not stated explicitly, but in specs we do use camel case convention. That's what I observed at least.